boards/riscv/icev_wireless/Kconfig.board

```diff
@@ -0,0 +1,7 @@
+# Copyright (c) 2022 Friedt Professional Engineering Services, Inc
+# SPDX-License-Identifier: Apache-2.0
+
+config BOARD_ICEV_WIRELESS
+	bool "ICE-V Wireless Board"
+	depends on SOC_ESP32C3
+	select CONSOLE_HAS_DRIVER
```

Comment:
feels like on odd place to have `select CONSOLE_HAS_DRIVER`

Reply:
Fixed!